src/modules/displaymanager/main.py

```diff
@@ -51,35 +54,46 @@
 
 
 def find_desktop_environment(root_mount_point):
-    """ Checks which desktop environment is currently installed.
+    """
+    Checks which desktop environment is currently installed.
 
     :param root_mount_point:
     :return:
     """
     for desktop_environment in desktop_environments:
-        if os.path.exists(
-            "{!s}{!s}".format(root_mount_point, desktop_environment.executable)) \
-            and os.path.exists("{!s}/usr/share/xsessions/{!s}.desktop".format(root_mount_point,
-                                                                              desktop_environment.desktop_file)):
+        if (os.path.exists("{!s}{!s}".format(
+                    root_mount_point, desktop_environment.executable
+                    )
+                ) and os.path.exists(
+                    "{!s}/usr/share/xsessions/{!s}.desktop".format(
+                        root_mount_point, desktop_environment.desktop_file
+                    )
+                )):
             return desktop_environment
-
     return None
 
 
 def have_dm(dm_name, root_mount_point):
-    """ Checks if display manager is properly installed.
+    """
+    Checks if display manager is properly installed.
 
     :param dm_name:
     :param root_mount_point:
     :return:
     """
     return os.path.exists(
-        "{!s}/usr/bin/{!s}".format(root_mount_point, dm_name)) or os.path.exists(
-        "{!s}/usr/sbin/{!s}".format(root_mount_point, dm_name))
+        "{!s}/usr/bin/{!s}".format(root_mount_point, dm_name)
+        ) or os.path.exists(
+            "{!s}/usr/sbin/{!s}".format(root_mount_point, dm_name)
+            )
 
```

Comment:
If pycodestyle agrees, I'd typically do this as 

```
bin_path = "{!s}/usr/bin/{!s}".format(root_mount_point, dm_name)
sbin_path = "{!s}/usr/sbin/{!s}".format(root_mount_point, dm_name)
return (
    os.path.exists(bin_path)
    or os.path.exists(sbin_path)
    )
```

That makes clearer what is being tested, exactly.